.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "lib/forge-std"]
```

Comment:
Is there no way to use `forge-std` without adding a submodule dependency? I always find them a real pain to work with, but personal preference. 

I'd probably just copy the `forge-std` contract over and it's dependencies

Reply:
Let me see if I can just remove this dependency, using forge-std caused hardhat compile to break due to remappings not working there.

Reply:
so this should be unused